locations/dict_parser.py

```diff
@@ -86,7 +86,7 @@ class DictParser:
         "yext-display-lng",
     ]
 
-    website_keys = ["url", "website", "permalink"]
+    website_keys = ["url", "website", "permalink", "storeUrl"]
```

Comment:
Yeah, we should handle `storeUrl`, but I'd prefer `store-url` to match the others.

```suggestion
    website_keys = ["url", "website", "permalink", "store-url"]
```